Fix/vue identicon attrs listeners

Ryan-Gordon opened the pull request:
Fixes vue-polkadot/vue-ui#23

Bumped the vue version and vue-template-compiler. 

Explanation of the fix: 
# Reproduce issue

Replace this line everywhere 
`import Identicon from '@polkadot/vue-identicon';`

with this line : 
`import Identicon from '@vue-polkadot/vue-identicon';`

This will then use Identicon from vue-polkadot and show the issue when you head to accounts/create

### Identify the issue
The issue was not anything to do with vue-identicon but in how it is being used in `vue-polkadot/apps`
In `vue-polkadot/apps` vue is pinned to version `2.6.10` but in the vue-identicon it is given an upwards pin of `^2.6.10`
This resulted in 2 defined versions of vue being installed and the cause of the issue.

### Verify the cause of the issue.
Before attempting a fix, replace `'@polkadot/vue-identicon';` with `'@vue-polkadot/vue-identicon'` as explained, install deps and serve. Then run `yarn why vue` and you will see it is being loaded with different versions in different places

### Suggested fixes
I see two fixes here. 
1. Update vue to 2.6.11 in `vue-polkadot/apps`
2. Update the vue dep in vue-polkdadot/apps to `^2.6.10` this will prevent double loading different versions of vue when using identicon in `vue-polkadot/apps`

I would recommend option 2 to prevent the issue creeping up again.

Ryan-Gordon committed:
refactor: replace @polkadot/vue-identicon with @vue-polkadot/vue-identicon everywhere

Ryan-Gordon committed:
fix: bump vue and vue-template-compiler

Bumped 1 version but it may be better to use this syntax going forward
`^2.6.10` which aligns with vue-identicon
Fixes vue-polkadot
/
vue-ui#23

Ryan-Gordon commented:
This issue took quite a bit of time to debug and understand so happy to see this is deployed well.

Ryan-Gordon commented:
Does the `size` parameter need changing ?

Ryan-Gordon commented:
@yangwao @vikiival Could I get a review on this ?

vikiival commented:
@Ryan-Gordon We will look at it! ❇️ 


> Does the `size` parameter need changing ?

Yes, parameter size has to be dynamic

Ryan-Gordon committed:
fix: add size Prop with default value

Only place remains unchanged is
2 components in TxSelect which has two values neither of which are 64 (56 vs 48). Set the default here to be the higher value, 56.

Ryan-Gordon commented:
@vikiival pushed another commit which uses size with a prop so it can be dynamic. Did a search and found a few places its used. All the places I found now have both a Prop `size` and a default value. 👍

yangwao commented:
I've just checked it out and looks perfect, great work @ryan-gordon 🙌

Can you rebase your PR against develop branch? 

I thought I've set default branch to develop here 😇

yangwao commented:
Seems it's working :) merging!
